app/queries/drug_stocks_query.rb

```diff
@@ -10,23 +12,18 @@ def initialize(facilities:, for_end_of_month:)
     @latest_drug_stocks = DrugStock.latest_for_facilities(@facilities, @for_end_of_month)
   end
 
-  attr_reader :for_end_of_month
-
-  def protocol_drugs_by_category
-    @protocol_drugs_by_category ||= @protocol.protocol_drugs
-      .where(stock_tracked: true)
-      .sort_by(&:sort_key)
-      .group_by(&:drug_category)
-      .sort_by { |(drug_category, _)| drug_category }
-      .to_h
-  end
+  attr_reader :for_end_of_month, :facilities
 
   def drug_stocks_report
     Rails.cache.fetch(drug_stocks_cache_key,
       expires_in: ENV.fetch("ANALYTICS_DASHBOARD_CACHE_TTL"),
       force: RequestStore.store[:bust_cache]) do
-      {all: drug_stock_totals,
-       facilities: drug_stock_report_for_facilities,
+      {total_patients: total_patients,
+       all_patient_days: all_patient_days,
+       all_drugs_in_stock: all_drugs_in_stock,
```

Comment:
Can these three attributes be named the way way as the three following ones, without the `by_facility_id` suffix? That would be very consistent and easy to correlate.

Reply:
Changed in dfe7952